README.md

```diff
@@ -166,13 +166,3 @@ The Dockerfile in this repo can be built locally using the following command:
 ```bash
 docker image build .  -t demo --build-arg ENVIRONMENT_CONFIG=local
 ```
-
-This will build the image with your `app-config.local.yaml` included this allows you to include any config you might want to have in place for local testing.
-
-Using `--progress=plain --no-cache` can also be helpful with testing changes.
-
-You can then run it with this command:
-
-```bash
-docker run -it -p 7007:7007 demo
-```
```

Comment:
The content being removed here is helpful, I do look at this from time to time when I do need to build the image locally. 👍